components/service/glean/src/main/java/mozilla/components/service/glean/scheduler/PingUploadWorker.kt

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean.scheduler
+
+import android.content.Context
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import mozilla.components.service.glean.Glean
+import mozilla.components.service.glean.net.HttpPingUploader
+
+/**
+ * This class is the worker class used by [WorkManager] to handle uploading the ping to the server.
+ */
+class PingUploadWorker(context: Context, params: WorkerParameters) : Worker(context, params) {
+    companion object {
+        internal const val PING_WORKER_TAG = "glean_ping_worker"
```

Comment:
nit: I assume this is unique enough. But just FYI: In other situations we use "mozac" (for **Moz**illa **A**ndroid **C**omponents) and the component name as prefix, e.g. something like `mozac_service_glean_...`

Reply:
I'd like to stay in sync with how other components are set up so I'll modify this to be more in line with that.